stl/inc/deque

```diff
@@ -555,6 +555,18 @@ public:
 
     _Deque_val() noexcept : _Map(), _Mapsize(0), _Myoff(0), _Mysize(0) {}
 
+#if _MSVC_STL_DESTRUCTOR_TOMBSTONES
+    ~_Deque_val() noexcept {

```

Comment:
Does the presence or absence of a non-trivial destructor count as ABI incompatible?

Reply:
Copy constructors, yes. (That happened in `tuple<>`.) Destructors, not to my knowledge. And the derived `deque` definitely has a non-trivial destructor.

Triviality does prevent us from messing with `optional<Trivial>` and `variant<AllTrivial>`.